<?xml version="1.0" encoding="ISO-8859-15"?>
<launch>

  <rosparam file="$(find semantic_octomap)/params/semantic_cloud.yaml" command="load"/>
  <!--<rosparam file="$(find semantic_octomap)/params/semantic_labels.yaml" command="load"/>-->
  <node pkg="semantic_octomap" type="semantic_sensor_node.py" name="semantic_cloud" output="screen">
    <param name="agent_name" value="" />
  </node>
  <rosparam file="$(find semantic_octomap)/params/octomap_generator.yaml" command="load"/>
  <node pkg="semantic_octomap" type="semantic_octomap_node" name="semantic_octomap" output="screen"/>


  <arg name="body_frame" default="semantic_segmentation_world"/>
  <arg name="optic_frame" default="camera_optic"/>
  <node pkg="tf2_ros" type="static_transform_publisher" name="body2camera" 
  args="0 0 0 -1.5708 0 -1.5708 $(arg body_frame) $(arg optic_frame)" />

</launch>
